cli/fuzzers/afl_symcc/build_from_args.cpp

```diff
@@ -0,0 +1,91 @@
+          "Specify PUT and args for PUT.")(
+          "frida",
+          po::value<bool>(&afl_options.frida_mode)
+              ->default_value(afl_options.frida_mode),
+          "Enable/disable frida mode. Default to false.")(
+          "symcc_target", po::value<std::string>(&symcc_options.target_path),
+          "Path to the target executable compiled by SymCC.")(
+          "symcc_freq", po::value<std::size_t>(&symcc_options.symcc_freq),
+          "SymCC execution frequency."
+          "If 0, SymCC is never executed."
+          "Otherwise, SymCC is executed if recent n local loop blocks didn't "
+          "change the corpus."
+          "Default 1.");
+
+  po::variables_map vm;
+  po::store(po::command_line_parser(fuzzer_args.argc, fuzzer_args.argv)
+                .options(fuzzer_desc)
+                .positional(pargs_desc)
+                .run(),
+            vm);
+  po::notify(vm);
+
+  if (global_options.help) {
+    fuzzuf::cli::fuzzer::afl::usage(fuzzer_desc);
+  }
+  if (!vm.count("symcc_target")) {
+    fuzzuf::cli::fuzzer::afl::usage(fuzzer_desc);
+  }
+  std::vector<std::string> symcc_args(pargs.begin(), pargs.end());
+  if (symcc_args.empty())
+    symcc_args.push_back(symcc_options.target_path);
+  else
+    symcc_args[0] = symcc_options.target_path;
+  const auto symcc_dir =
+      fs::absolute(fs::path(global_options.out_dir) / "symcc");
+  return std::unique_ptr<fuzzuf::algorithm::afl_symcc::AFLSymCCFuzzer>(
+      new fuzzuf::algorithm::afl_symcc::AFLSymCCFuzzer(
+          fuzzuf::cli::fuzzer::afl::BuildFuzzer<
+              fuzzuf::algorithm::afl_symcc::AFLFuzzerTemplate<
+                  fuzzuf::algorithm::afl::AFLState>,
+              fuzzuf::algorithm::afl_symcc::AFLFuzzerTemplate<
+                  fuzzuf::algorithm::afl::AFLState>,
+              fuzzuf::executor::AFLExecutorInterface>(fuzzer_args.argv[0],
+                                                      fuzzer_desc, afl_options,
+                                                      pargs, global_options),
+          std::move(symcc_options),
+          std::shared_ptr<fuzzuf::executor::AFLSymCCExecutorInterface>(
+              new fuzzuf::executor::AFLSymCCExecutorInterface(
+                  std::shared_ptr<NativeLinuxExecutor>(new NativeLinuxExecutor(
+                      std::move(symcc_args),
+                      global_options.exec_timelimit_ms.value_or(
+                          fuzzuf::algorithm::afl::option::GetExecTimeout<
+                              fuzzuf::algorithm::afl::option::AFLTag>()),
+                      global_options.exec_memlimit.value_or(
+                          fuzzuf::algorithm::afl::option::GetMemLimit<
+                              fuzzuf::algorithm::afl::option::AFLTag>()),
+                      false, fs::path(global_options.out_dir) / "cur_input", 0,
+                      0, false, {"SYMCC_OUTPUT_DIR=" + symcc_dir.string()},
+                      {symcc_dir}))))));
```

Comment:
This is just a tiny problem, but here are too many namespaces and thus this part is a bit difficult to read. 
I think there are some omittable namespace specifiers like `fuzzuf`, so probably they should be omitted.